Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindowUtils.cs

```diff
@@ -14,10 +15,13 @@ internal class InputActionsEditorWindowUtils
         public static void SaveAsset(SerializedObject serializedAsset)
         {
             var asset = (InputActionAsset)serializedAsset.targetObject;
-            // for the global actions asset: save differently (as it is a yaml file and not a json)
+            // For project-wide actions asset save works differently. The asset is in YAML format, not JSON.
             if (asset.name == ProjectWideActionsAsset.kAssetName)
             {
+                ProjectWideActionsAsset.UpdateInputActionReferences();
                 AssetDatabase.SaveAssets();
+
+                var existingReferences = InputActionImporter.LoadInputActionReferencesFromAsset(ProjectWideActionsAsset.GetOrCreate()).ToList();
```

Comment:
Shouldn't it be `asset` instead of `ProjectWideActionsAsset.GetOrCreate()`?
Or did you really need to create another asset instance for some reason here?